src/services/FirebaseService.js

```diff
@@ -0,0 +1,88 @@
+import Firebase from 'firebase';
+import * as ActionTypes from '../constants/actionTypes';
+
+export const FirebaseService = {
+  _ref: null,
+  _dispatcher: null,
+
+  createTask(task) {
+    this._ref
+      .child('teams/fwk-int/tasks/')
+      .push()
+      .set({
+        sectionId: task.sectionId,
+        content: task.content,
+      }, (err) => {
+        if (!err) {
+          this._dispatcher.dispatch({
+            type: 'FIREBASE_TASK_CREATED_OK',
+            payload: null,
+          });
+        }
+      });
+  },
+
+  updateTask({id, sectionId, content}) {
+    this._ref
+      .child('teams/fwk-int/tasks/')
+      .child(id)
+      .set({
+        sectionId,
+        content,
+      }, (err) => {
+        if (!err) {
+          this._dispatcher.dispatch({
+            type: 'FIREBASE_TASK_UPDATED_OK',
+            payload: null,
+          });
+        }
+      });
+  },
+  archiveTask(task) {
+    return new Promise((resolve, reject) => {
+      this._ref
+        .child('teams/fwk-int/tasks/')
+        .child(task.id)
+        .set(null, (err) => {
+          if (!err) {
+            this._ref
+              .child('teams/fwk-int/archived-tasks/')
```

Comment:
maybe camel case in name is better, since we are using for example 'sectionId'?

archivedTasks